gitbook-docs/evm/verifier.md

```diff
@@ -0,0 +1,194 @@
+# Placeholder verifier
+
+Placeholder proof system and its associated verifier offer an efficient solution
+for zero-knowledge proof verification.
+Whether you are building complex decentralized applications or need to verify
+the correctness of computations in a trustless manner, Placeholder provides
+flexibility and modularity to meet your specific requirements.
+This documentation serves as a guide to help you make the most of this powerful tool.
+
+## What is Placeholder
+
+[Placeholder proof system](https://nil.foundation/blog/post/placeholder-proofsystem)
+is a cutting-edge proof system with IVC (Incrementally Verifiable Computations)
+and custom gates.
+It's built to enable zero-knowledge functionality and secure interactions
+on the Ethereum blockchain and other blockchain networks.
+Placeholder serves a zero-knowledge succinct non-interactive argument of knowledge
+(zkSNARK) based on PlonK-style arithmetization.
+Its modularity allows configuring and adapting it for various use cases,
+each with distinct security requirements.
+
+For technical details and implementation insights, you can explore the source code
+in [Crypto3's GitHub repository](https://github.com/nilfoundation/crypto3).
+
+## What is Placeholder verifier
+
+[Placeholder verifier](https://github.com/NilFoundation/evm-placeholder-verification/)
+is an essential component that enables the in-EVM verification of zero-knowledge proofs
+generated using the Placeholder proof system.
+It's a smart contract application that can be deployed on different Ethereum networks,
+including local, test, and main networks.
+
+Currently, it's deployed on the Sepolia network
+([contract](https://sepolia.etherscan.io/address/0x489dbc0762b3d9bd9843db11eecd2a177d84ba2b)).
+You can check the full code and tests
+[on GitHub](https://github.com/NilFoundation/evm-placeholder-verification/).
+
+## Installing verifier as a dependency
+
+The verifier is published as an npm package, so you can easily add it as a dependency
+for your project.
+Open a terminal and run the following command from your project's home:
+
+```bash
+npm install @nilfoundation/evm-placeholder-verification@1.1.1
+```
+
+Alternatively, you can manually add it to your project's `package.json` file
+by including the following line:
+
+```json
+"@nilfoundation/evm-placeholder-verification": "1.1.1"
+```
+
+## On-chain verification
+
+For on-chain verification of zero-knowledge proofs via Placeholder verifier
+import the verifier contract in your Solidity code as shown below:
+
+```
```

Comment:
I believe this code is `solidity`.

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting
https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml